content/blog/getting-started-with-deno/puppeteer-with-deno.md

```diff
@@ -0,0 +1,217 @@
+
+Out of this experience's learnings, and following the evolution of JavaScript over the last 10 years, Ryan created Deno, which is, as the documentation says:
+
+> A JavaScript/TypeScript runtime with secure defaults and a great developer experience.
+
+We previously explored some of Deno's premises and how it addresses specific Node.js problems in another article named [Adventures in deno land](https://alexandrempsantos.com/adventures-in-deno-land/). This time we're here for a series or articles that will explore different Deno features.
+
+Today we'll explore `deno-puppeteer`, a port of [puppeteer]() to deno. We'll demonstrate how Deno can make it even simpler to write *puppeteer* scripts and applications.
+
+## Deno with Puppeteer
+
+Puppeteer was a changing piece of technology. It enabled developers to directly use JavaScript to control their headless browser of choice (Chrome & Firefox) without all the burden that it used to be.
+
+Together with great documentation and an engaged community, Puppeteer has been one of the best solutions when it comes to writing applications connecting with headless browsers.
+
+A few months ago, [Luca Casonato](https://twitter.com/lcasdev), one of Deno's core team members, ported Puppeteer into Deno.
+
+https://twitter.com/lcasdev/status/1344279906809741312
+
+This is, by itself, an interesting topic to explore: how much did the code have to change to get it working in Deno. But that's not what we're doing here today.
+
+As Deno gains its place among script tools, being able to use *puppeteer* is another interesting step taken. With `deno-puppeteer`, users  can now benefit from the ease of use of Deno while writing *puppeteer* scripts.
+
+In this blogpost we will build a CLI utility that demonstrates that.
+
+The full code is available [here](https://github.com/asantos00/deno-website-diff).
+## Writing a Puppeteer script
+
+The objective of the CLI utility we'll build is to check a website for visual changes in different resolutions.
+
+This tool will compare the website with the last time it checked, creating an image with the difference. It can work as a QA assurance tool, something that runs after deploying a website to confirm that it is working fine in different resolutions.
+
+To achieve this, we'll use a couple community packages and tools. Some are functions from Deno's standard library, others are just Node.js packages
+
+- pngjs - PNG encoder/decoder in JS
+- pixelmatch - A JS image comparison library
+- Deno file-system APIs (`readFile` and `writeFile`)
+
+We'll use [jspm](https://jspm.org/) to make sure `pngjs` and `pixelmatch` (both Node.js packages) are **ES6 module compatible**. This will make sure they work on Deno (yes, Deno is fully ES6 compatible!).
```

Comment:
```suggestion
We'll use [jspm](https://jspm.org/) to make sure `pngjs` and `pixelmatch` (both Node.js packages) are **ES6 module compatible**. This will make sure that they work on Deno (yes, Deno is fully ES6 compatible!).
```